src/api/cpp/GravityConfigParser.cpp

```diff
@@ -26,6 +26,9 @@
 #include <map>
 #include <iostream>
 
+#ifdef min
```

Comment:
If these lines no longer have any value, please delete them.  If there's a decent chance they might be needed again, can you add a comment explaining why?

Reply:
Nevermind, misread what this was doing